go.mod

```diff
@@ -14,7 +14,7 @@ require (
 	github.com/containerd/containerd v1.7.21
 	github.com/containerd/containerd/api v1.7.19
 	github.com/containerd/fifo v1.1.0
-	github.com/containerd/nri v0.7.0
+	github.com/containerd/nri v0.7.1-0.20241031201336-6d486acd3f71
```

Comment:
Do we have a release to pin to or can we cut one?

Reply:
Samuel says is going to cut one https://github.com/containerd/containerd/pull/10921#issuecomment-2451072751 , I was just sending the PR to check the CI and get the functionality on both runtimes at the same time

Reply:
We can wait for the tag, I just leave this open in the meantime if you don't mind

Reply:
Sounds great, thank you!